ingest.py

```diff
@@ -1,316 +0,0 @@
-import re
-from functools import partial
-from itertools import izip
-from more_itertools import unique_everseen
-from urlparse import urlparse
-
-import arrow
-import requests
-from bs4 import BeautifulSoup
-from funcy import notnone, walk_keys, without
-
-from app import db
-from utils import find_player_name, multiple_replace
-
-
-_WINLOSS_REGEX = re.compile('.*?\((.*?)\)')
-
-
-class Ingester(object):
-    @staticmethod
-    def _date_conversion(text):
-        """
-        Converts text to a date.
-
-        :param text: String to convert.
-        """
-        return arrow.get(text).datetime
-
-    @staticmethod
-    def _home_conversion(text):
-        """
-        Converts text to a boolean stating whether a game was a home game.
-
-        :param text: String to convert, one of "" or "@".
-        """
-        return text != '@'
-
-    @staticmethod
-    def _winloss_conversion(text):
-        """
-        Converts the text to a margin of victory.
-
-        :param text: String to find margin within, must contain the margin
-            within parentheses.
-        """
-        return float(_WINLOSS_REGEX.match(text).group(1))
-
-    @staticmethod
-    def _percent_conversion(text):
-        """
-        Percentages are not converted, so this returns none.
-
-        :param text: Original string.
-        """
-        return None
```

Comment:
This is kind of odd. Since you call .get() when you grab these from the dictionary, wouldn't it just be easier to not define a number for _percent_conversions()? That way you save the confusion and simplify the code a bit.